ansible/README.md

```diff
@@ -149,6 +149,11 @@ You need to run `initdb` on cloudant **only once** per cloudant database to init
 **Hint:** The `initdb.yml` playbook will only initialize your database if it is not initialized already, else it will skip initialization steps.
 
 
+### Verification after Deployment
+
+After a successful deployment you can use the `wsk` CLI (located in the /bin folder of the repository) to verify that OpenWhisk is operable.
+See [README](https://github.com/openwhisk/openwhisk/blob/master/README.md)
```

Comment:
See [main readme](https://github.com/openwhisk/openwhisk/blob/master/README.md) for instructions on how to setup and use `wsk`.